lib/application.ts

```diff
@@ -285,11 +285,11 @@ export class SNApplication {
   }
 
   public onStart() {
-
+    console.error('onStart not implemented');
   }
 
   public onLaunch() {
-
+    console.error('onLaunch not implemented');
   }
```

Comment:
Clients don't have to implement onStart or onLaunch for SNJS to work properly. Why log it?

Reply:
Because of this error: https://github.com/typescript-eslint/typescript-eslint/blob/v3.10.1/packages/eslint-plugin/docs/rules/no-empty-function.md

I'm open to suggestion on how to change this

Reply:
I would silence the warning with a comment explaining that it's an optional override